api/response/schema_query.py

```diff
@@ -88,38 +86,37 @@ def get_all_reponses(self, info):
         rooms = RoomModel.query.all()
         for room in rooms:
             room_name = room.name
-            total_response = ResponseModel.query.filter_by(
-                room_id=room.id).count()
             room_response = Response.get_query(info).filter_by(room_id=room.id)
+            total_response = room_response.count()
             all_responses = Query.get_room_response(self, room_response)
```

Comment:
This change was made because `room_response.count()` is equal to `total_response=ResponseModel.query.filter_by(	room_id=room.id).count()`.